### PR TITLE
YT-11639 Changes to add custom subject and body for email

### PR DESCRIPTION
**Pull request by shamsultech89**

### CODE REVIEW

File: lib/docusign_rest/client.rb [hunk 1]
```diff
@@ -345,7 +345,12 @@ def get_signers(signers, options={})
         end
 
         if signer[:email_notification]
-          doc_signer[:emailNotification] = signer[:email_notification]
```
Comment (chiragsinghal) [inline lib/docusign_rest/client.rb:348]: if we already have a node for this, why do we need to modify the logic here?

Comment (chiragsinghal) [reply lib/docusign_rest/client.rb:348]: and if we have to make changes, then can we not simply use transform keys to change to camel case?

**Comment by chiragsinghal** [comment]
@pramodtech @gsivananda please review

File: lib/docusign_rest/client.rb (hunk 1 above)
Comment (gsivananda) [reply lib/docusign_rest/client.rb:348]: I think we were not using this node earlier as we haven't added `email_notification` key while building signers on Manager side. Now @shamsultech89 using this key while building signer. Generally, we were following ruby style for naming keys on Manager side and here we are checking and assigning it to camel case.

Comment (shamsultech89) [reply lib/docusign_rest/client.rb:348]: > and if we have to make changes, then can we not simply use transform keys to change to camel case?

Transform keys are available in ruby 2.5 and this gem is using ruby 2.0. So we can create a ticket to update the gem.

**Review by pramodtech — APPROVED** [review]

**Pull request merged**